templates/profile/card$.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix foaf: <http://xmlns.com/foaf/0.1/>.
+@prefix schema: <http://schema.org/>.
+
+<>
+    a foaf:PersonalProfileDocument ;
+    foaf:maker <{{webId}}> ;
+    foaf:primaryTopic <{{webId}}> .
+
+<{{webId}}>
+    a foaf:Person ;
+    a schema:Person ;
+
+    foaf:name "{{name}}".
```

Comment:
```suggestion
    a foaf:Person, schema:Person ;
    foaf:name "{{name}}".
```

Not sure why `schema:Person` there if we're not using other Schema.org attributes.

Reply:
Because I copied part of the default templates from NSS to have some kind of default file in there 😅